lib/widgets/gridcards.dart

```diff
@@ -0,0 +1,334 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:sochem/utils/constants.dart';
+
+class HomeScreenGrid extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        Row(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            Padding(
+              padding:
+                  const EdgeInsets.only(top: 20.0, left: 15.0, right: 15.0),
+              child: Container(
+                height: MediaQuery.of(context).size.height * 0.15,
+                width: MediaQuery.of(context).size.width * 0.40,
+                decoration: BoxDecoration(
```

Comment:
Same as above, multiple usage of MediaQuery should be avoided, instead use contants defined at the start of file.

Reply:
Done